src/Paths.js

```diff
@@ -1,20 +1,35 @@
 let path = require('path');
 
-module.exports = {
+class Paths {
+    constructor() {
+        this.rootPath = path.resolve(__dirname, '../../../');
+    }
+
+    /**
+     * Set the root path to resolve webpack.mix.js
+     * @param {string} path
+     */
+    setRootPath(path) {
+        this.rootPath = path;
+
+        return this;
+    }
+
     /**
      * Determine the path to the user's webpack.mix.js file.
      */
-    mix: function () {
+    mix() {
         return this.root('webpack.mix');
-    },
-
+    }
 
     /**
      * Determine the project root.
      *
+     * @param {string|null} rootPath
```

Comment:
Haha yes sorry I forgot to remove it when I refactored this to a class. I corrected it and also added a missing one in `Mix.detectHotReload`